helm-charts/ingress-controller/README.md

```diff
@@ -29,40 +21,24 @@ helm install ingress-easegress -n ingress-easegress ./helm-charts/ingress-contro
   --set ingressClass.name=test-eg \
   --set controller.name=test-eg \
   --set 'controller.namespaces={ingress-easegress, default}'
-
-# install cluster of 3 primary and 2 secondary Easegress ingress instances
-helm install ingress-easegress -n ingress-easegress ./helm-charts/ingress-controller \
-  --set cluster.primaryReplicas=3 \
-  --set cluster.secondaryReplicas=2
-
-# install using persistentVolume on node with hostname "hostname-xyz"
-# to support recovery when pod crashes
-helm install ingress-easegress -n ingress-easegress ./helm-charts/ingress-controller \
-  --set cluster.volumeType=persistentVolume \
-  --set 'cluster.nodeHostnames={hostname-xyz}'
 ```
 
+You can reference [this example](/doc/reference/ingresscontroller.md#create-backend-service--kubernetes-ingress) for creating Kubernetes Ingress rules.
+
 ## Uninstall
 
 ```shell
 helm uninstall ingress-easegress -n ingress-easegress
-
-#sometimes helm does not delete pvc and pv. Delete manually each pvc.
-kubectl delete pvc easegress-pv-ingress-easegress-0 -n ingress-easegress
-# same for easegress-pv-ingress-easegress-i...n
 ```
 
 ## Parameters
 
-The following table lists the configurable parameters of the MegaEase Easegress Helm installation.
+The following table lists the configurable parameters of the Ingress Controller Helm installation.
 
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
-| service.nodePort | int | `30080` | nodePort for easegress service. |
-| service.adminPort | int | `31255` | nodePort for egctl access. |
-| cluster.primaryReplicas | int | `1` | number of easegress service that persists cluster data to disk |
-| cluster.volumeType | string | `emptyDir` | `emptyDir`: use pods internal filesystem that is not persisted when pod crashes. `persistentVolume`, create as many persistenVolumes and persistentVolumeClaims as there are nodeHostnames.
-| cluster.nodeHostnames | list | `[]` | nodeHostnames are hostnames of VMs/Kubernetes nodes. Only used when `volumeType: persistentVolume`. Note that this require nodes to be static. |
-| secondaryReplicas | int | `0` | number of easegress service that not persists cluster data to disk. |
+| service.nodePort | int | `30080` | nodePort for Easegress Ingress Controller. |
+| service.adminPort | int | `31255` | nodePort for egctl admin access. |
```

Comment:
Do we need to expose this port?
The ingress controller instances are all standalone, the admin command sent to this port will be load-balanced to different instances, which results in the pods of the service in an inconsistent state.
But I don't think there's much need for ingress controller to process an admin command.

Reply:
Good point, we don't really need admin access here. I'll remove this.